src/dxvk/dxvk_graphics_state.h

```diff
@@ -447,11 +451,21 @@ namespace dxvk {
       return VkLogicOp(m_logicOp);
     }
 
+    bool hazardColor() const {
+      return !!m_hazardColor;
+    }
+
+    bool hazardDepth() const {
+      return !!m_hazardDepth;
+    }
+
   private:
 
     uint16_t m_enableLogicOp          : 1;
     uint16_t m_logicOp                : 4;
-    uint16_t m_reserved               : 11;
+    uint16_t m_hazardColor            : 1;
+    uint16_t m_hazardDepth            : 1;
```

Comment:
`uint16_t feedbackLoop : 2` and then add a function to get/set it as `VkImageAspectFlags`